package.json

```diff
@@ -91,17 +91,12 @@
       "versions": {
         "^1.0.0": "consumeStatusBar"
       }
-    },
-    "tree-view": {
-      "versions": {
-        "^1.0.0": "consumeTreeView"
-      }
```

Comment:
Any idea what this was for?

Reply:
No clue, but it isn't being used since that function isn't in main (or anywhere from a file search)

Reply:
Looking at the git-blame looks like you added this to add an import notebook command when right-clicking on `.ipynb` files in the tree-view, not sure where this code was moved, but it has been implemented somewhere else as it still works in this PR

Reply:
This is because we just retrieved file path from class name, not thought via valid tree-view API here: https://github.com/nteract/hydrogen/blame/renovate/mobx-react-6.x/lib/import-notebook.js#L29

I think the current implementation works thus we can leave this as is.